composeApp/src/commonMain/kotlin/com/majotyler/hiittimer/presentation/TimerViewModel.kt

```diff
@@ -0,0 +1,29 @@
+package com.majotyler.hiittimer.presentation
+
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+
+class TimerViewModel : ViewModel() {
+    private val _state = MutableStateFlow(value = TimerViewState(seconds = 0))
+    val state = _state.asStateFlow()
+
+    fun onEvent(event: TimerViewEvent) {
```

Comment:
Notice how this function is public, but the others are `private`. This is intentional, so `TimerScreen` can only see `onEvent`. This is clean architecture. It makes code hard to break.